packages/flutter/lib/src/services/font_loader.dart

```diff
@@ -62,7 +66,19 @@ class FontLoader {
             (Uint8List list) => loadFont(list, family)
         )
     );
-    return Future.wait(loadFutures.toList());
+    return Future.wait(loadFutures.toList()).then((List<void> futures) async {
+      if (ServicesBinding.instance != null) {
+        final BasicMessageChannel<dynamic> systemChannel = SystemChannels.system;
+        await systemChannel.binaryMessenger.handlePlatformMessage(
```

Comment:
It is odd to call this from here...

Reply:
the other way to do it is to make ServiceBinding.instance.handleSystemMessage public. I think they are almost equally odd..

Reply:
Yeah, we should find a way that isn't odd. Communicating via message channels within the framework does seem like a hack.